Remove atk-bridge support.

atk-bridge doesn't seem useful for now in Buildroot and requires to
add two new packages just for it: at-spi2-core and at-spi2-atk.

Signed-off-by: Hadrien Boutteville <hadrien.boutteville@gmail.com>

--- a/configure.ac
+++ b/configure.ac
@@ -1349,11 +1349,7 @@
 # Check for Accessibility Toolkit flags
 ########################################

-if test x$enable_x11_backend = xyes; then
-   ATK_PACKAGES="atk atk-bridge-2.0"
-else
-   ATK_PACKAGES="atk"
-fi
+ATK_PACKAGES="atk"

 PKG_CHECK_MODULES(ATK, $ATK_PACKAGES)

--- a/gtk/a11y/gtkaccessibility.c
+++ b/gtk/a11y/gtkaccessibility.c
@@ -37,10 +37,6 @@
 #include <gtk/gtkcombobox.h>
 #include <gtk/gtkaccessible.h>

-#ifdef GDK_WINDOWING_X11
-#include <atk-bridge.h>
-#endif
-
 static gboolean gail_focus_watcher      (GSignalInvocationHint *ihint,
                                          guint                  n_param_values,
                                          const GValue          *param_values,
@@ -987,9 +983,5 @@
   _gtk_accessibility_override_atk_util ();
   do_window_event_initialization ();

-#ifdef GDK_WINDOWING_X11
-  atk_bridge_adaptor_init (NULL, NULL);
-#endif
-
   atk_misc_instance = g_object_new (GTK_TYPE_MISC_IMPL, NULL);
 }
